velox/exec/TreeOfLosers.h

```diff
@@ -44,6 +44,13 @@ class MergeStream {
   // Returns true if the first element of 'this' is less than the first element
   // of 'other'. hasData() must be true of 'this' and 'other'.
   virtual bool operator<(const MergeStream& other) const = 0;
+
+  // Returns < 0 if 'this' is < 'other, '0' if equl and > 0 otherwise. This is
+  // not required for TreeOfLosers::next() but is required for
+  // TreeOfLosers::nextWithEquals().
+  virtual int32_t compare(const MergeStream& other) const {
```

Comment:
Would it be cleaner to have only one compare, e.g. replace operator< with this more generic compare method?